components/vf-section-header/vf-section-header.scss

```diff
@@ -10,6 +10,28 @@
  */
 
 .vf-section-header__heading {
-  @include set-type(text-heading--3);
-  margin: 0;
+  @include set-type(text-heading--3, $custom-margin-bottom: 24px);
+}
+a.vf-section-header__heading {
```

Comment:
why not `.vf-section-header__heading--is-link`? or something similar?

Reply:
I was thinking this when writing it. 

I guess, as we should be moving things on embl.org to using the templates rather than having it as copy/pasted HTML like it currently is - we can do this.

I wonder if `.is-link` would be better and more 'atomic' but then I have a feeling we will be hit by 'edge cases' anyhow.

Will update to `vf-section-header__heading--is-link`and push -- although that is quite the mouthful :sob: